### ISSUE TITLE
lab 3 error in get_nlcd()

### ISSUE THREAD

**Issue by billspat** [SpaCE-Lab-MSU/MSUGradSpatialEcology#4]
this line ( around 183): 

```R
hood_nlcd<-get_nlcd(
   template = hood,
   label="Hood",
   year = 2016,
   dataset = "Land_Cover",
   extraction.dir = paste0(data_path, "/FedData/"),
  )
```

Error in get_nlcd_tile(template = template, year = year, dataset = dataset,  : 

  Parameter 'dataset' must be one of **'landcover',** 'impervious', or 'canopy'.

I'm rendering from the root folder, not the 'docs' folder so the students may have a different version?

**Comment by plzmsu** [comment]
Yes this is if you install FedData directly from CRAN. I ran into issues with it with that install for some reason, then found a whole bunch of posts about how they suggest you install from GitHub - see the installation instructions near the top of the Lab. One of the differences is the terms the commands are referencing (e.g., landcover vs. Land.Cover ... annoying).
